test_conformance/vulkan/vulkan_interop_common/vulkan_wrapper_types.hpp

```diff
@@ -0,0 +1,390 @@
+{
+    VULKAN_MEMORY_HEAP_FLAG_NONE         = 0,
+    VULKAN_MEMORY_HEAP_FLAG_DEVICE_LOCAL = VK_MEMORY_HEAP_DEVICE_LOCAL_BIT
+};
+
+enum VulkanExternalMemoryHandleType
+{
+    VULKAN_EXTERNAL_MEMORY_HANDLE_TYPE_NONE                = 0,
+    VULKAN_EXTERNAL_MEMORY_HANDLE_TYPE_OPAQUE_FD           = VK_EXTERNAL_MEMORY_HANDLE_TYPE_OPAQUE_FD_BIT_KHR,
+    VULKAN_EXTERNAL_MEMORY_HANDLE_TYPE_OPAQUE_WIN32_NT     = VK_EXTERNAL_MEMORY_HANDLE_TYPE_OPAQUE_WIN32_BIT_KHR,
+    VULKAN_EXTERNAL_MEMORY_HANDLE_TYPE_OPAQUE_WIN32_KMT    = VK_EXTERNAL_MEMORY_HANDLE_TYPE_OPAQUE_WIN32_KMT_BIT_KHR,
+    VULKAN_EXTERNAL_MEMORY_HANDLE_TYPE_OPAQUE_WIN32_NT_KMT = VK_EXTERNAL_MEMORY_HANDLE_TYPE_OPAQUE_WIN32_BIT_KHR | VK_EXTERNAL_MEMORY_HANDLE_TYPE_OPAQUE_WIN32_KMT_BIT_KHR
+};
+
+enum VulkanExternalSemaphoreHandleType
+{
+    VULKAN_EXTERNAL_SEMAPHORE_HANDLE_TYPE_NONE                = 0,
+    VULKAN_EXTERNAL_SEMAPHORE_HANDLE_TYPE_OPAQUE_FD           = VK_EXTERNAL_SEMAPHORE_HANDLE_TYPE_OPAQUE_FD_BIT_KHR,
+    VULKAN_EXTERNAL_SEMAPHORE_HANDLE_TYPE_OPAQUE_WIN32_NT     = VK_EXTERNAL_SEMAPHORE_HANDLE_TYPE_OPAQUE_WIN32_BIT_KHR,
+    VULKAN_EXTERNAL_SEMAPHORE_HANDLE_TYPE_OPAQUE_WIN32_KMT    = VK_EXTERNAL_SEMAPHORE_HANDLE_TYPE_OPAQUE_WIN32_KMT_BIT_KHR,
+    VULKAN_EXTERNAL_SEMAPHORE_HANDLE_TYPE_OPAQUE_WIN32_NT_KMT = VK_EXTERNAL_SEMAPHORE_HANDLE_TYPE_OPAQUE_WIN32_BIT_KHR | VK_EXTERNAL_SEMAPHORE_HANDLE_TYPE_OPAQUE_WIN32_KMT_BIT_KHR
+};
+
+enum VulkanBufferUsage
+{
+    VULKAN_BUFFER_USAGE_TRANSFER_SRC                    = VK_BUFFER_USAGE_TRANSFER_SRC_BIT,
+    VULKAN_BUFFER_USAGE_TRANSFER_DST                    = VK_BUFFER_USAGE_TRANSFER_DST_BIT,
+    VULKAN_BUFFER_USAGE_UNIFORM_TEXEL_BUFFER            = VK_BUFFER_USAGE_UNIFORM_TEXEL_BUFFER_BIT,
+    VULKAN_BUFFER_USAGE_STORAGE_TEXEL_BUFFER            = VK_BUFFER_USAGE_STORAGE_TEXEL_BUFFER_BIT,
+    VULKAN_BUFFER_USAGE_UNIFORM_BUFFER                  = VK_BUFFER_USAGE_UNIFORM_BUFFER_BIT,
+    VULKAN_BUFFER_USAGE_STORAGE_BUFFER                  = VK_BUFFER_USAGE_STORAGE_BUFFER_BIT,
+    VULKAN_BUFFER_USAGE_INDEX_BUFFER                    = VK_BUFFER_USAGE_INDEX_BUFFER_BIT,
+    VULKAN_BUFFER_USAGE_VERTEX_BUFFER                   = VK_BUFFER_USAGE_VERTEX_BUFFER_BIT,
+    VULKAN_BUFFER_USAGE_INDIRECT_BUFFER                 = VK_BUFFER_USAGE_INDIRECT_BUFFER_BIT,
+    VULKAN_BUFFER_USAGE_STORAGE_BUFFER_TRANSFER_SRC_DST = VK_BUFFER_USAGE_STORAGE_BUFFER_BIT | VK_BUFFER_USAGE_TRANSFER_SRC_BIT | VK_BUFFER_USAGE_TRANSFER_DST_BIT,
+    VULKAN_BUFFER_USAGE_UNIFORM_BUFFER_TRANSFER_SRC_DST = VK_BUFFER_USAGE_UNIFORM_BUFFER_BIT | VK_BUFFER_USAGE_TRANSFER_SRC_BIT | VK_BUFFER_USAGE_TRANSFER_DST_BIT,
+};
+
+enum VulkanSharingMode
+{
+    VULKAN_SHARING_MODE_EXCLUSIVE  = VK_SHARING_MODE_EXCLUSIVE,
+    VULKAN_SHARING_MODE_CONCURRENT = VK_SHARING_MODE_CONCURRENT
+};
+
+enum VulkanImageType
+{
+    VULKAN_IMAGE_TYPE_1D = VK_IMAGE_TYPE_1D,
+    VULKAN_IMAGE_TYPE_2D = VK_IMAGE_TYPE_2D,
+    VULKAN_IMAGE_TYPE_3D = VK_IMAGE_TYPE_3D
+};
+
+enum VulkanFormat

```

Comment:
What's the purpose of these duplicate definitions? These will be a maintenance problem.

Reply:
Seem like just convention with renaming. Will consider as part of cleanup and reusing standard Vulkan headers.